.github/ISSUE_TEMPLATE/release_card_template

```diff
@@ -0,0 +1,33 @@
+---
+name: Release card template
+about: Template to use when build cards are done and we're ready to release a new or updated component / template 
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Summary:** Release of the XXXX component. 
+When build dev/design and content are all synced and requirements and decisions are documented, we're ready to release component on all touch points. 
+
+*Ddefinition of done:** 
+- [ ]Merge figma work to main and publish library
+- [ ]Component package is released
+- [ ]Publish new component on documentation website 
+- [ ]Related updates to other component pages are made, as needed
+- [ ]Inform engagement team of upcoming release 
+- [ ]Publish a release info on slack 
```

Comment:
- [ ] Done when design has merged the component in figma
- [ ] Done when development has merged the PR r
- [ ] Done when development has released the package(s)
- [ ] Done when the website has been updated with documentation for the guidance
- [ ] Done when a release message has been posted on slack
- [ ] Done when engagement team is informed of the release